Library/Homebrew/formula_installer.rb

```diff
@@ -379,6 +379,7 @@ def check_conflicts
   # Compute and collect the dependencies needed by the formula currently
   # being installed.
   def compute_dependencies
+    install_cask_requirements
```

Comment:
I don't think installing casks should happen in a method named `compute_dependencies`. Additionally, requirements already can have `cask`s so it would be good to be able to map those casks onto casks to install.

Reply:
Yes, I will move that the `install` method.
And I'm sorry, but does mapping the casks mean to store the name of casks to install into the variable `@cask` of a requirement instance? This is already implemented like this, in the new subclass `CaskRequirement`. @MikeMcQuaid

Reply:
> And I'm sorry, but does mapping the casks mean to store the name of casks to install into the variable @cask of a requirement instance?

I'm not sure I totally understand but yes it means making use of the already existing `cask` in requirements e.g. https://github.com/Homebrew/brew/blob/master/Library/Homebrew/requirements/x11_requirement.rb#L8

Reply:
Yes, that's similar. `CaskRequirement` will set `@cask` according to the initialization argument.